Description: Fix 'bellow' typo, replace with 'below'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: http://gmt.soest.hawaii.edu/issues/795

--- a/doc/rst/source/grdvolume.rst
+++ b/doc/rst/source/grdvolume.rst
@@ -54,7 +54,7 @@ Optional Arguments
     height of the entire grid]. The area is measured in the plane of the contour.
     The **Cr** form on the other hand computes volume between the grid
     surface and the plans defined by *low* and *high*. Note that this is an
-    *outside* volume whilst the other forms compute an *inside* (bellow the surface)
+    *outside* volume whilst the other forms compute an *inside* (below the surface)
     area volume. Use this form to compute for example the volume of water
     between two contours.
 **-L**\ *base*
--- a/src/gmt_customio.c
+++ b/src/gmt_customio.c
@@ -1099,7 +1099,7 @@ struct srf_header6 {	/* Surfer 6 file he
 
    Note: I had significant troubles to be able to read correctly the Surfer 7 format.
    In its basic and mostly used form (that is, without break-lines info) what we normally
-   call a header, can be described by the srf_header7 structure bellow (but including
+   call a header, can be described by the srf_header7 structure below (but including
    the three commented lines). This would make that the header is composed of 2 char[4] and
    and 5 ints followed by doubles. The problem was that after the ints the doubles were not
    read correctly. It looked like everything was displaced by 4 bytes.
--- a/src/gmtswitch.bat
+++ b/src/gmtswitch.bat
@@ -37,7 +37,7 @@ REM	The second mode is temporary because
 REM	prepended to previous path via a shell command line. This modification disappears
 REM	when the shell cmd window where it was executes is deleted.
 REM
-REM	It is the user responsibility to set the contents of the G32_32 to G5_64 bellow
+REM	It is the user responsibility to set the contents of the G32_32 to G5_64 below
 REM	to valid paths where the binaries of the different GMT versions are installed
 REM	Note that it's not mandatory to have all four of them in you computer. For the
 REM	ones you do not have just let them pointing to nothing e.g.
--- a/src/potential/gmtgravmag3d.c
+++ b/src/potential/gmtgravmag3d.c
@@ -747,7 +747,7 @@ int read_xyz (struct GMT_CTRL *GMT, stru
 	
 	if (Ctrl->box.is_geog) {	/* take a first read just to compute the central longitude */
 		while (fgets (line, GMT_LEN256, fp)) {
-			sscanf (line, "%lg %lg", &in[0], &in[1]); /* A test on file integrity will be done bellow */
+			sscanf (line, "%lg %lg", &in[0], &in[1]); /* A test on file integrity will be done below */
 			x_min = (float)MIN(in[0], x_min);	x_max = (float)MAX(in[0], x_max);
 			y_min = (float)MIN(in[1], y_min);	y_max = (float)MAX(in[1], y_max);
 		}
