Description: Disable non-free triangle source use.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -231,15 +231,15 @@ if (DO_API_TESTS)
 	set (GMT_DEMOS_SRCS testapi.c testpsl.c testgmt5.c testgmtio.c testgrdio.c testio.c testapiconv.c example1.c testapi_matrix.c testapi_vector.c test_JL.c test_walter.c)
 endif (DO_API_TESTS)
 
-if (NOT LICENSE_RESTRICTED) # off
-	# enable Shewchuk's triangle routine
-	set (GMT_TRIANGULATE "Shewchuk" PARENT_SCOPE)
-	set (GMT_TRIANGLE_SRCS triangle.c triangle.h)
-	list (APPEND GMT_EXTRA_LICENSE_FILES ${CMAKE_CURRENT_DIR}/README.TRIANGLE)
-else (NOT LICENSE_RESTRICTED) # on
+#if (NOT LICENSE_RESTRICTED) # off
+#	# enable Shewchuk's triangle routine
+#	set (GMT_TRIANGULATE "Shewchuk" PARENT_SCOPE)
+#	set (GMT_TRIANGLE_SRCS triangle.c triangle.h)
+#	list (APPEND GMT_EXTRA_LICENSE_FILES ${CMAKE_CURRENT_DIR}/README.TRIANGLE)
+#else (NOT LICENSE_RESTRICTED) # on
 	# disable Shewchuk's triangle routine
 	set (GMT_TRIANGULATE "Watson" PARENT_SCOPE)
-endif (NOT LICENSE_RESTRICTED)
+#endif (NOT LICENSE_RESTRICTED)
 
 # Note: Developers can set EXTRA_MODULES in ConfigUser.cmake to test new modules
 set (GMT_PROGS_SRCS blockmean.c blockmedian.c blockmode.c filter1d.c
@@ -276,7 +276,7 @@ set (GMT_LIB_SRCS block_subs.h common_by
 	gmt_plot.h gmt_project.h gmt_prototypes.h gmt_psl.h gmt_shore.h
 	gmt_symbol.h gmt_synopsis.h gmt_texture.h gmt_time.h
 	gmt_types.h gmt_unique.h gmt_version.h gshhg_version.h postscriptlight.h
-	s_rint.h triangle.h gmt.h gmt_resources.h gmt_private.h
+	s_rint.h gmt.h gmt_resources.h gmt_private.h
 	gmt_error_codes.h gmt_api.c gmt_bcr.c gmt_calclock.c gmt_cdf.c gmt_compat.c
 	gmt_customio.c gmt_dcw.c gmt_error.c gmt_fft.c gmt_grdio.c gmt_io.c
 	gmt_memory.c gmt_nc.c gmt_notposix.c gmt_proj.c gmt_shore.c gmt_sph.c
@@ -378,11 +378,6 @@ configure_file (gmt-config.in gmt-config
 # gmtprogram
 set (GMT_PROGRAM ${GMT_SOURCE_DIR}/src/gmtprogram.c)
 
-# extra ugly definitions for triangle
-set_source_files_properties (triangle.c
-	PROPERTIES
-	COMPILE_DEFINITIONS "NO_TIMER;TRILIBRARY;REDUCED;CDT_ONLY;ANSI_DECLARATORS")
-
 # libpslib
 add_library (pslib
 	${GMT_GEN_HDRS}
