Index: GMT4.2.0/Makefile
===================================================================
--- GMT4.2.0.orig/Makefile	2007-04-02 22:38:49.000000000 +0200
+++ GMT4.2.0/Makefile	2007-04-16 23:02:28.632988058 +0200
@@ -65,7 +65,7 @@
 #	!! STOP EDITING HERE, THE REST IS FIXED !!
 #-------------------------------------------------------------------------------
 
-SUPPL	=	dbase gshhs imgsrc meca mex mgd77 mgg misc segyprogs spotter x2sys x_system xgrid
+SUPPL	=	dbase gshhs imgsrc meca mgd77 mgg misc segyprogs spotter x2sys x_system xgrid
 SUPPL_M	=	dbase imgsrc meca mgd77 mgg misc segyprogs spotter x2sys x_system
 
 all:		gmt suppl
