Description: Use ucontext from FreeBSD for GNU/kFreeBSD too.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://gmt.soest.hawaii.edu/issues/720

--- a/src/common_sighandler.c
+++ b/src/common_sighandler.c
@@ -64,7 +64,7 @@ void backtrace_symbols_fd(void *const *b
 # else
 #  define UC_IP(uc) ((void *) (uc)->uc_mcontext->ss.eip)
 # endif
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
 # ifdef __x86_64__
 #  define UC_IP(uc) ((void *) (uc)->uc_mcontext.mc_rip)
 # elif defined( __arm__)
