Description: Fix 'optionaly' typo, replace with 'optionally'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: http://gmt.soest.hawaii.edu/issues/758

--- a/doc/rst/source/supplements/potential/gmtgravmag3d.rst
+++ b/doc/rst/source/supplements/potential/gmtgravmag3d.rst
@@ -81,7 +81,7 @@ Optional Arguments
 **-T**\ [[*d*]\ *xyz_file*/*vert_file*\ [*/m*]]\|[*r\|s*]\ *raw_file*]
     Give either names of xyz[m] and *vertex* files or of a *raw* or
     *stl* file defining a close surface. In the first case append a *d*
-    imediatly after **-T** and optionaly a */m* after the vertex file
+    imediatly after **-T** and optionally a */m* after the vertex file
     name. In the second case append a *r* or a *s* imediatly after
     **-T** and before the file name. A *vertex* file is a file with N
     rows (one per triangle) and 3 columns with integers defining the
--- a/src/potential/gmtgravmag3d.c
+++ b/src/potential/gmtgravmag3d.c
@@ -181,7 +181,7 @@ int GMT_gmtgravmag3d_usage (struct GMTAP
 	GMT_Option (API, "R");
 	GMT_Message (API, GMT_TIME_NONE, "\t-S search radius in km\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t-T Give either names of xyz[m] and vertex files or of a file defining a close surface.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   In the first case append an 'd' imediatly after -T and optionaly a /m after the vertex file name.\n");
+	GMT_Message (API, GMT_TIME_NONE, "\t   In the first case append an 'd' imediatly after -T and optionally a /m after the vertex file name.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   In the second case append an 'r' or a 's' imediatly after -T and before the file name.\n");
 	GMT_Message (API, GMT_TIME_NONE, "\t   'r' and 's' stand for files in raw (x1 y1 z1 x2 ... z3) or STL format.\n");
 	GMT_Option (API, "V");
--- a/src/potential/gravfft.c
+++ b/src/potential/gravfft.c
@@ -438,7 +438,7 @@ int GMT_gravfft_usage (struct GMTAPI_CTR
 	GMT_Message (API, GMT_TIME_NONE,"\t   water, all in SI units. Give average mantle depth via -Z\n");
 	GMT_Message (API, GMT_TIME_NONE,"\t   If the elastic thickness is > 1e10 it will be interpreted as the\n");
 	GMT_Message (API, GMT_TIME_NONE,"\t   flexural rigidity (by default it is computed from Te and Young modulus).\n");
-	GMT_Message (API, GMT_TIME_NONE,"\t   Optionaly, append +m to write a grid with the Moho's geopotential effect\n");
+	GMT_Message (API, GMT_TIME_NONE,"\t   Optionally, append +m to write a grid with the Moho's geopotential effect\n");
 	GMT_Message (API, GMT_TIME_NONE,"\t   (see -F) from model selected by -T.\n");
 	GMT_Message (API, GMT_TIME_NONE,"\t-Z Specify Moho [and swell] average compensation depths.\n");
 	GMT_Option (API, "V");
